docs(guide):  updated readme guide

superbgv opened the pull request:
Fix the Readme error:
Old Readme point to add `DEFAULT_CONFIG_URL_PREFIX` & `DEFAULT_LOCATION_URL_PREFIX`
to system environment, but it should be added in to project’s global`gradle.properties`

superbgv committed:
docs(guide):  updated readme guide

Fix the Readme error:
Old Readme point to add `DEFAULT_CONFIG_URL_PREFIX` & `DEFAULT_LOCATION_URL_PREFIX`
  to system environment, but it should be added in to project’s global`gradle.properties`

NemoOudeis commented on README.md:
```diff
@@ -25,13 +25,20 @@ The modules are:
 & git submodule update
-$ export DEFAULT_CONFIG_URL_PREFIX="url to your configuration server"
-$ export DEFAULT_LOCATION_URL_PREFIX="url to your location server"
 $ ./gradlew Runtime:assemble Plugin:assemble
```
Instead of adding another section to the readme i would elaborate on the build command

```diff
-$ export DEFAULT_CONFIG_URL_PREFIX="url to your configuration server"		
-$ export DEFAULT_LOCATION_URL_PREFIX="url to your location server"
-$ ./gradlew Runtime:assemble Plugin:assemble
+ $ ./gradlew Runtime:assemble Plugin:assemble \
+ $   -PDEFAULT_CONFIG_URL_PREFIX="url to your configuration server" \
+ $   -PDEFAULT_LOCATION_URL_PREFIX ="url to your location server"
```

see https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_properties_and_system_properties

My rationale is: whenever I see a readme that asks me to set something up first I'm more inclined to not even try using that thing.

superbgv replied:
make sense, will update the PR.

superbgv committed:
docs(guide):  updated readme guide

Fix the Readme error:
Old Readme point to add `DEFAULT_CONFIG_URL_PREFIX` & `DEFAULT_LOCATION_URL_PREFIX`
  to system environment, but these should be provided as gradle project properties
